### PR TITLE
[MIG] partner_risk_insurance: Migration to 14.0 

### PR DESCRIPTION
**Pull request by Zar21**
Standard migration to 14.0

### CODE REVIEW

File: partner_risk_insurance/__manifest__.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 {
     "name": "Partner Risk Insurance",
-    "version": "13.0.1.1.1",
+    "version": "14.0.1.1.1",
```
Comment (pedrobaeza) [inline partner_risk_insurance/__manifest__.py]: This should be 14.0.1.0.0

**Comment by francesco-ooops** [comment]
Hi @Zar21 , any plans to finish this one? 

Thanks!

**Comment by francesco-ooops** [comment]
thanks @Zar21 !

**Pull request merged**